operators-manual/companion-web.md

```diff
@@ -243,6 +243,9 @@ Steps to manually update via the [Git page](http://192.168.2.2:2770/git):
 Starting from companion 0.0.19, a MAVLink REST API ([http://192.168.2.2:4777/mavlink](http://192.168.2.2:4777/mavlink)) endpoint is available.
 It provides access to the last MAVLink messages sent by the vehicle in [json format](https://www.json.org).
 
+> **Note** This feature requires an endpoint `--out udpin:0.0.0.0:14660` at the companion [MavProxy page](#mavproxy). Companion 0.0.19 and newer ship with that as a default, but if you updated from a previous version you may need to [**Restore Default Options**](#mavproxy).
+
```

Comment:
It's missing a warning that mavlink2 is necessary for the functionality of the REST API, the serial parameter need to be configured in vehicle side. 

There is also an unnecessary extra empty line.

Reply:
> It's missing a warning that mavlink2 is necessary for the functionality of the REST API

Ha, I didn't remember this! updating